wolfssl/wolfcrypt/settings.h

```diff
@@ -3059,6 +3059,9 @@ extern void uITRON4_free(void *p) ;
         #undef HAVE_ECC_KEY_EXPORT
         #define HAVE_ECC_KEY_EXPORT
     #endif
+    #ifndef WOLFSSL_NO_VALIDATE_ECC_IMPORT
```

Comment:
needs to be
```
    #if !defined(WOLFSSL_NO_VALIDATE_ECC_IMPORT) && \
        !defined(WOLFSSL_VALIDATE_ECC_IMPORT)
```
to avoid `error: ... redefined`.